fabric-rendering-fluids-v1/src/main/java/net/fabricmc/fabric/api/client/render/fluid/v1/FluidTextureRegistry.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.render.fluid.v1;
+
+import net.minecraft.screen.PlayerScreenHandler;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.api.event.client.ClientSpriteRegistryCallback;
+
+/**
+ * Implements the registration for fluid textures.
+ */
+public interface FluidTextureRegistry {
```

Comment:
In C# this would have been done with a static class, but java has not static classes...

Maybe this way?

``` java
/**
 * Implements the registration for fluid textures.
 */
public final class FluidTextureRegistry {
	private FluidTextureRegistry() {
	}

	/**
	 * Registers the fluid textures.
	 *
	 * @param textures Textures to register.
	 */
	public static void register(Identifier... textures) {
		ClientSpriteRegistryCallback.event(PlayerScreenHandler.BLOCK_ATLAS_TEXTURE).register((atlasTexture, registry) -> {
			for (Identifier texture : textures) {
				registry.register(texture);
			}
		});
	}
}
```

Reply:
Yeah, that's how you'd do it in java